app/routes/index.js

```diff
@@ -5,22 +5,22 @@ export default Ember.Route.extend({
     model() {
         return Ember.RSVP.hash({
             theDate: new Date(),
-
             preprints: this.store.findAll('preprint'),
             subjects: this.store.findAll('taxonomy')
         });
     },
     actions: {
+        // TODO: properly transfer subject to discover route
         goToSubject(sub) {
-            this.transitionTo('browse', { queryParams: { subject: sub } });
+            this.transitionTo('discover', { queryParams: { subject: sub } });
```

Comment:
This was just to emulate ember-share, which uses 'discover' instead of 'browse'. I also figured this would avoid potential merge conflicts along the way since I'd be starting from scratch in a separate route.